### PR TITLE
Meta: define merge policy and add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by annevk**
Helps with https://github.com/whatwg/meta/issues/65.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,11 @@ End tags may be included (if done consistently) and attributes may be quoted (us
 though the prevelant theme is to omit end tags and not quote attributes (unless they contain a
 space).
 
+### Merge policy
+
+If you can commit to this repository, see the
+[maintainer guidelines](https://github.com/whatwg/meta/blob/master/MAINTAINERS.md).
```
Comment (snuggs) [inline README.md]: Do we want `WHATWG maintainer guidelines` for consistency?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Please see the [contributor guidelines](https://github.com/whatwg/meta/blob/master/CONTRIBUTING.md).
```
Comment (snuggs) [inline CONTRIBUTING.md]: Great!

File: README.md (hunk 1 above)
Comment (annevk) [reply README.md]: Hmm maybe. If we did that we should say "WHATWG Maintainer Guidelines" I suppose, just like "WHATWG Code of Conduct". I guess I could update any outstanding PRs and fix it for the remainder if something else "Meta" needs updating in the future.

Comment (snuggs) [reply README.md]: ... if something else "Meta" needs updating in the future. - @annevk 

I see what you did there ;-)

**Pull request merged**